src/py/covidr/app.py

```diff
@@ -1,25 +1,104 @@
 import os
 from pathlib import Path
+from functools import wraps
+
+from dotenv import find_dotenv, load_dotenv
+from flask import Flask, send_from_directory, redirect, session, url_for
+from authlib.integrations.flask_client import OAuth
+from urllib.parse import urlencode
 
 from covidr.config import DevelopmentConfig
-from flask import Flask, send_from_directory
+
+
+ENV_FILE = find_dotenv()
+if ENV_FILE:
+    load_dotenv(ENV_FILE)
```

Comment:
Could you add documentation for what needs to be in the `.env` file and where it's supposed to be located?

Reply:
yes that is a very good idea

Reply:
do you think this would be better in the README or in the backend docs?

Reply:
I imagine that README is a top-level directory plus anything that caters to a broad audience.  So in this case, the backend docs makes more sense.